doc/classes/SpriteFrames.xml

```diff
@@ -59,6 +64,7 @@
 			<argument index="0" name="anim" type="String">
 			</argument>
 			<description>
+				The animation's FPS.
```

Comment:
Expanding the abbreviation to just "frames per second" would be clearer. (Not that people don't know what FPS is, just it is an abbreviation too much)

Reply:
Yeah, I debated that one. Expanded.